test/lazy/test_ts_opinfo.py

```diff
@@ -0,0 +1,161 @@
+from torch.testing._internal.common_methods_invocations import op_db
+from torch.testing._internal.common_device_type import ops, instantiate_device_type_tests
+
+import torch._lazy
+import torch._lazy.metrics
+import torch._lazy.ts_backend
+import itertools
+import yaml
+import os
+import pathlib
+
+
+torch._lazy.ts_backend.init()
+
+def get_test_device():
+    return 'cuda' if 'LTC_TS_CUDA' in os.environ else 'cpu'
+
+def remove_suffixes(l):
+    return [x.split(".")[0] for x in l]
+
+def init_lists():
+    path_to_script = pathlib.Path(os.path.abspath(os.path.dirname(__file__)))
+    TS_NATIVE_FUNCTIONS_PATH = path_to_script.parent.parent / "aten/src/ATen/native/ts_native_functions.yaml"
+    with open(TS_NATIVE_FUNCTIONS_PATH) as f:
+        yaml_ts = yaml.load(f, yaml.Loader)
+    LAZY_OPS_LIST = set(remove_suffixes(itertools.chain(yaml_ts["full_codegen"], yaml_ts["supported"], yaml_ts["autograd"])))
+    FALLBACK_LIST = set(["clamp"])
+    SKIP_RUNTIME_ERROR_LIST = set([
+        'index_select',  # Empty output_sizes is not supported
+        'clone',  # is clone decomposed?
+    ])
+    SKIP_INCORRECT_RESULTS_LIST = set([
+        'squeeze',  # Value out of range
+        't',  # Value out of range
+        'transpose',  # Value out of range
+        'bernoulli',  # incorrect results
+        'pow', # incorrect results
+    ])
+
+    return (LAZY_OPS_LIST, FALLBACK_LIST, SKIP_RUNTIME_ERROR_LIST, SKIP_INCORRECT_RESULTS_LIST)
+
+(LAZY_OPS_LIST, FALLBACK_LIST, SKIP_RUNTIME_ERROR_LIST, SKIP_INCORRECT_RESULTS_LIST) = init_lists()
+
+torch.manual_seed(42)
+
+class TestLazyTensor(JitTestCase):
+    def testConvolutionBackward(self):
+        def clone_move(t):
+            dev = 'lazy'
+            copy_t = t.detach().clone().requires_grad_(True).to(device=dev)
+            return copy_t
+
+        test_device = get_test_device()
+        inp = torch.rand(1, 3, 128, 128, device=test_device, requires_grad=True)
+        inp_copy = clone_move(inp)
+        grad = torch.rand(1, 32, 121, 121, device=test_device)  # no requires_grad
+        grad_copy = clone_move(grad)
+        weight = torch.rand(32, 3, 8, 8, device=test_device, requires_grad=True)
+        weight_copy = clone_move(weight)
+        bias = torch.rand(32, device=test_device, requires_grad=True)
+        bias_copy = clone_move(bias)
+
+        # run eager
+        conv_out = torch.nn.functional.conv2d(inp, weight, bias)
+        (inp_grad, weight_grad, bias_grad) = torch.autograd.grad([conv_out], [inp, weight, bias], [grad])
+
+        # run lazy
+        conv_copy_out = torch.nn.functional.conv2d(inp_copy, weight_copy, bias_copy)
+        (inp_copy_grad, weight_copy_grad, bias_copy_grad) = torch.autograd.grad(
+            [conv_copy_out], [inp_copy, weight_copy, bias_copy], [grad_copy])
+
+        # TODO(whc) do we want to change how this API is exposed?
+        # jit_graph = torch._lazy._get_ltc_tensors_backend([bias_copy_grad])
```

Comment:
@Krovatkin I assume this line is not important.  (i also assume you wrote it, maybe @desertfire did?)

Reply:
yes you can remove it. When core folks fixed convolution to flow through one op, @desertfire removed the graph checks since convolutions become just another op